postgres/postgres_test.go

```diff
@@ -43,14 +46,47 @@ func TestGetInstance(t *testing.T) {
 		ds, err := NewDatastore(db)
 		So(err, ShouldBeNil)
 		mock.ExpectPrepare(findInstanceSQL).ExpectQuery().
-			WithArgs(sqlmock.AnyArg()).WillReturnRows(sqlmock.NewRows([]string{"job"}).
-			AddRow(jsonContent))
-		state, err := ds.GetInstance("any")
+			WithArgs(sqlmock.AnyArg()).WillReturnRows(sqlmock.NewRows([]string{"jobId","instance"}).
+			AddRow( jsonContent, "1"))
+		state, err := ds.GetInstance("http://localhost:80", "any")
 		So(err, ShouldBeNil)
 		So(state.State, ShouldEqual, "Created")
 	})
 }
 
+func TestGetJobs(t *testing.T) {
+	t.Parallel()
+	Convey("When get jobs is called, a list of jobs are returned", t, func() {
+		jsonContent := "{ \"state\":\"Created\"}"
+		mock, db := NewSQLMockWithSQLStatements()
+		ds, err := NewDatastore(db)
+		So(err, ShouldBeNil)
+		mock.ExpectPrepare(getJobsSQL).ExpectQuery().
+			WillReturnRows(sqlmock.NewRows([]string{"jobid", "instanceid","json"}).
+			AddRow(1, 1, jsonContent))
+		jobs, err := ds.GetJobs("localhost", []string{})
+		So(err, ShouldBeNil)
+		So(jobs[0].State, ShouldEqual, "Created")
```

Comment:
Should we be checking 'mock.ExpectationsWereMet()' ? https://github.com/DATA-DOG/go-sqlmock#tests-with-sqlmock